app/pages/index.js

```diff
@@ -295,41 +308,44 @@ const Home = ({ randomContributors }) => {
+                  La autenticación integrada es muy fácil y muy segura. Funciona con cualquier
+                  proovedor de identidad, incluidos el nombre de usuario y contraseña de tu hosting
+                  y terceros como Auth0.
                 </FeatureIconTitle>
-                <FeatureIconTitle icon="database" title="Database Agnostic">
-                  You can use any database you want. Prisma 2 is the default database client, but
-                  you can remove that and use anything else like Fauna or DynamoDB.
+                <FeatureIconTitle icon="database" title="Cualquier base de datos">
+                  Puedes utilizar la base de datos que quieras. Prisma 2 es el cliente de base de
+                  datos predeterminado, pero puedes eliminarlo y usar cualquier otro como Fauna o
+                  DynamoDB.
                 </FeatureIconTitle>
-                <FeatureIconTitle icon="fileCode" title="Recipes">
-                  One command to install code and/or packages into your blitz app. Examples: `blitz
-                  install tailwind` or `blitz install chakra-ui`. Recipes can be created by anyone.
+                <FeatureIconTitle icon="fileCode" title="Recetas">
+                  Un comando para instalar el código y/o paquetes de tu aplicación blitz. Ejemplos:
+                  `blitz install tailwind` o `blitz install chakra-ui`. Cualquiera puede crear una
+                  "receta".
                 </FeatureIconTitle>
-                <FeatureIconTitle icon="plugin" title="Backend Architecture">
-                  Blitz is set up for server intensive tasks like sending emails, cron jobs,
-                  background processing, generating PDFs, etc. Currently we have minimal backend
-                  guides, but are working on developing more patterns and adding more docs.
+                <FeatureIconTitle icon="plugin" title="Arquitectura Backend">
+                  Blitz está configurado para tareas que requieren un uso intensivo del servidor,
+                  como enviar emails, cronjobs, procesamiento en segundo plano, generar archivos
+                  PDF, etc. Actualmente tenemos pocas guías de backend, pero estamos trabajando para
+                  desarrollar más patrones y agregar más documentos.
                 </FeatureIconTitle>
-                <FeatureIconTitle icon="typescript" title="First Class Typescript Support">
-                  Blitz is built with Typescript and the Blitz data layer is fully end-to-end
-                  typesafe. All types are completely static without needing a separate type
-                  generation process!
+                <FeatureIconTitle icon="typescript" title="Soporte a TypeScript de primera clase">
+                  Blitz está construido con TypeScript y la capa de datos de Blitz es totalmente
+                  end-to-end y segura por tipos. ¡Todos los tipos son completamente estáticos sin
```

Comment:
"es totalmente end-to-end y segura por tipos" -> "está completamente asegurada por _tipos_"